doc/getting_started/_index.md

```diff
@@ -193,14 +196,26 @@ You can download and build Verible from scratch as explained on the [Verible Git
 But since this requires the Bazel build system the recommendation is to download and install a pre-built binary as described below.
 
 Go to [this page](https://github.com/google/verible/releases) and download the correct binary archive for your machine.
-The example below is for Ubuntu 18.04:
+
+The example below is for Ubuntu 20.04:
+
+```
+export VERIBLE_VERSION=v0.0-2479-g92928558
```

Comment:
```suggestion
export VERIBLE_VERSION={{< tool_version "verible" >}}
```
I'd prefer to use the generated variable here so that this doesn't get out of date. I've used the current min version (`v0.0-2135-gb534c1fe`) on Ubuntu 20.04.x.

Reply:
I'd guessed (perhaps wrongly?) that the magic 'tool_version' picks up whatever is presently installed on the machine(s) that serve up the web documentation; is that not the case?
My understanding is that CI runs on Ubuntu 18.04, and my concern was that Ubuntu 20.04 would/may require a different version of Verible. Have now altered as requested and trust that version and OS build are orthogonal.

Reply:
> I'd guessed (perhaps wrongly?) that the magic 'tool_version' picks up whatever is presently installed on the machine(s) that serve up the web documentation; is that not the case?

I believe the version is taken from this file: `https://github.com/lowRISC/opentitan/blob/master/tool_requirements.py`

> My understanding is that CI runs on Ubuntu 18.04, and my concern was that Ubuntu 20.04 would/may require a different version of Verible. Have now altered as requested and trust that version and OS build are orthogonal.

They should be (hopefully!) be orthogonal... In any case we'll need to drop support for Ubuntu 18.04 in the near future (it is reaching EOL) and we'll be back to only supporting a single OS when that happens.

Reply:
@alees24 can you make this change? Once that is done I think we can merge this.

Reply:
Sorry, I did make that change at one point, but git and I got in a mess rebasing (my previous use of git has been simpler/more restricted) and the change must have been dropped in a reversion.